.github/workflows/docs.yaml

```
@@ -36,46 +40,36 @@ jobs:
 
   docs_build:
     name: Build Documentation
+    needs: [pixi_lock]
     runs-on: 'macos-latest'
     timeout-minutes: 120
     env:
       DISPLAY: ":99.0"
+      VERSION: ${{ inputs.version }}
     steps:
+      - name: Validate version
+        if: ${{ inputs.target == 'main' }}
+        run: |
+          if [[ -z "$VERSION" || ! "$VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "❌ 'version' is required and must match format X.X.X (e.g. 0.11.1) when target is 'main'"
+            exit 1
+          fi
```

Adding this here to fail fast.